libraries/Ethernet/src/ETH.cpp

```diff
@@ -1031,6 +1049,28 @@ void ETHClass::printInfo(Print & out){
     out.print(dnsIP());
     out.println();
 
+    static const char * types[] = { "UNKNOWN", "GLOBAL", "LINK_LOCAL", "SITE_LOCAL", "UNIQUE_LOCAL", "IPV4_MAPPED_IPV6" };
+    esp_ip6_addr_t if_ip6[CONFIG_LWIP_IPV6_NUM_ADDRESSES];
+    int v6addrs = esp_netif_get_all_ip6(_esp_netif, if_ip6);
+    for (int i = 0; i < v6addrs; ++i){
+        out.print("      ");
+        out.print("inet6 ");
+        IPAddress(IPv6, (const uint8_t *)if_ip6[i].addr, if_ip6[i].zone).printTo(out, true);
+        out.print(" type ");
+        out.print(types[esp_netif_ip6_get_addr_type(&if_ip6[i])]);
+        out.println();
+    }
+
+    // out.print("      ");
+    // out.print("inet6 ");
+    // localIPv6().printTo(out);
+    // out.println();
+
+    // out.print("      ");
+    // out.print("inet6 ");
+    // globalIPv6().printTo(out);
+    // out.println();
```

Comment:
Is this leftover prints from debugging ? IMO they should be removed or converted to `log_v`

Reply:
same, this code will not even be part of ETH anymore, but is here to poke me to implement it properly in the overhaul.